src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -267,10 +268,23 @@ void getHearingPriority(String isAdjournCase, String isUrgentCase, String expect
 
     @DisplayName("getNumberOfPhysicalAttendees Test")
     @Test
-    void getNumberOfPhysicalAttendees() {
-        Number result = HearingsDetailsMapping.getNumberOfPhysicalAttendees();
-
-        assertNull(result);
+    void shouldGetNumberOfPhysicalAttendees() {
```

Comment:
No offence but if you have to mock out this many objects you know there is too much going on - refactor / raise as TD